src/Observable.ts

```diff
@@ -89,7 +90,7 @@ export class Observable<T> implements CoreOperators<T>  {
     let subscriber: Subscriber<T>;
 
     if (observerOrNext && typeof observerOrNext === 'object') {
-      if (observerOrNext instanceof Subscriber || observerOrNext instanceof Subject) {
+      if (observerOrNext instanceof Subscriber) { //|| observerOrNext instanceof Subject) {
```

Comment:
This is part where regresses https://github.com/ReactiveX/RxJS/pull/863, ambient declaration of `Subject` cannot check its type via `instanceof` since it's interface.